Author: Daniel Baumann <daniel.baumann@progress-technologies.net>
Description:
 Update manpage to match source code regarding mapfile parameter, thanks to
 Jim Paris <jim@jtan.com> (Closes: #606267).

diff -Naurp tftp-hpa.orig/tftpd/tftpd.8.in tftp-hpa/tftpd/tftpd.8.in
--- tftp-hpa.orig/tftpd/tftpd.8.in	2010-07-10 14:05:18.284928211 +0200
+++ tftp-hpa/tftpd/tftpd.8.in	2010-12-24 17:39:16.308629646 +0100
@@ -150,7 +150,7 @@ or
 .B utimeout
 option is negotiated.  The default is 1000000 (1 second.)
 .TP
-\fB\-\-mapfile\fP \fIremap-file\fP, \fB\-m\fP \fIremap-file\fP
+\fB\-\-map-file\fP \fIremap-file\fP, \fB\-m\fP \fIremap-file\fP
 Specify the use of filename remapping.  The
 .I remap-file
 is a file containing the remapping rules.  See the section on filename
@@ -233,7 +233,7 @@ option, but crash with an error if they
 accepted by the server.
 .SH "FILENAME REMAPPING"
 The
-.B \-\-mapfile
+.B \-\-map-file
 option specifies a file which contains filename remapping rules.  Each
 non-comment line (comments begin with hash marks,
 .BR # )
@@ -385,7 +385,7 @@ flag is used to set up a chroot() enviro
 once a connection has been set up.
 .PP
 Finally, the filename remapping
-.RB ( \-\-mapfile
+.RB ( \-\-map-file
 flag) support can be used to provide a limited amount of additional
 access control.
 .SH "CONFORMING TO"
